custom/icds_reports/management/commands/queue_bihar_child_delivery_forms.py

```diff
@@ -0,0 +1,45 @@
+from datetime import date
+from django.core.management.base import BaseCommand
+from corehq.apps.userreports.tasks import _get_config_by_id
+from corehq.apps.userreports.util import get_table_name
+from corehq.apps.userreports.models import AsyncIndicator
+from django.db import connections
+
+from dimagi.utils.chunked import chunked
+
+
+class Command(BaseCommand):
+    help = "Rebuild Bihar delivery forms"
+
+    def handle(self, *args, **kwargs):
+        delivery_config = _get_config_by_id('static-icds-cas-static-child_delivery_forms')
+        table_name = get_table_name('icds-cas', 'static-child_delivery_forms')
+        # sort by supervisor_id and doc_id to improve the performance, sorting is needed to resume the queueing
+        # if it fails in between.
+        query = """
+            select supervisor_id, doc_id from "{}"
+            where state_id='f9b47ea2ee2d8a02acddeeb491d3e175'
```

Comment:
shouldn't this be an input argument?

Also if we are filtering by state_id the query will go to every Citus shard. It would be more efficient to get the list of supervisor IDs and query form them in chunks.

Reply:
Hey, Wont it be slower if I will hit the database multiple times for different chunks?

Reply:
I don't think so no - it may even be faster since it's doing less able to filter more efficiently

Reply:
I am printing the last supervisor_id below. Also I will fetch the supervisors in order so I will know what to fetch the records next from.